SoC/gd32vf103/Board/gd32vf103c_longan_nano/Include/gd32vf103c_longan_nano.h

```diff
@@ -48,6 +48,9 @@ typedef enum
     LED1 = 0,
     LED2 = 1,
     LED3 = 2,
+	LEDG = 0,
```

Comment:
Format code alignment here.

Reply:
The following two lines also need be aligned.

Reply:
Ok. Should be fixed in the last commit.